### DISCUSSION TITLE
Categories found in google but not mapped in the script

### DISCUSSION THREAD

**Discussion by Jer0** [omkarcloud/google-maps-scraper#100] [General]
Hello

I was wondering if it is possible to add new categories in the current mapping, right now I am trying to search for certain businesses based on a category that I cannot find in that list, this list can be expanded, is there somewhere in the Google documentation where all the categories are listed? categories they currently have?

Thank you!!

**Comment by Chetan11-dev** [comment]
There is no official Google Documentation for the categories, I found them via third party sources. For time being you can use simple strings for absent categories like. 
```python
Gmaps.places(queries, category_in=["dental clinic", "dental laboratory"])
```
